manifest/manifest.go

```diff
@@ -54,7 +54,7 @@ func GuessMIMEType(manifest []byte) string {
 	}
 
 	switch meta.MediaType {
-	case DockerV2Schema2MediaType, DockerV2ListMediaType, imgspecv1.MediaTypeImageManifest, imgspecv1.MediaTypeImageManifestList: // A recognized type.
+	case DockerV2Schema2MediaType, DockerV2ListMediaType, imgspecv1.MediaTypeImageManifest: // A recognized type.
```

Comment:
Yeah; when OCI has removed the `MediaType` field, we should have something like this PR whether or not we hide the build breakage by vendoring and old version.

(Or I guess we _could_ leave this in, with hard-coded strings, just to be able to deal with some existing pre-1.0 files.)